bin/cli.js

```diff
@@ -46,8 +48,8 @@ var defaultYargsInstance = yargs
   ].join('\n')))
```

Comment:
For future reference, in order for this to happen, there are two prerequisites:

- `/tasks/javascript.js`: remove all `require()`s from the exported function, so as to ensure that requiring the plugins involves only the cost of parsing the files - all requiring by the plugins themselves should be done within the their `onInit()` and `onRun()` hooks. This is already the case for the other tasks, just this one to go.
- `task-yargs`: Needs a new feature to enable definition of a `yargs` instance that is not attached to any sub-task (here known as `defaultYargsInstance`)